src/app/pcasts/dao/users_dao.py

```diff
@@ -84,7 +84,9 @@ def is_following_user(my_id, their_id):
 
 def search_users(search_name, offset, max_search, user_id):
   possible_users = User.query.filter \
-      (User.username.like('%' + search_name + '%')) \
+      (User.username.like('%' + search_name + '%') |
+       User.first_name.like('%' + search_name + '%') |
+       User.last_name.like('%' + search_name + '%')) \
```

Comment:
Assuming that search does not return duplicates(if your name is bob and you username is bob you return 2 bobs) it's good

Reply:
I don't think it will, just `SELECT * FROM users WHERE CONDITION1 OR CONDITION2 ....;`.  Just more possibilities for a hit.